acoustid/future/fingerprint.py

```
@@ -0,0 +1,56 @@
+
+    Returns:
+        Tuple containing a list of integer hash values and the version of the fingerprint.
+
+    The function decompresses the zstd data, unpacks the bytes into integers,
+    and reconstructs the original hash values by reversing the XOR differences.
+    """
+
+    try:
+        data = zstd.decompress(data)
+    except zstd.Error as e:
+        raise ValueError(f"Failed to decompress fingerprint: {e}") from e
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)